### PR TITLE
feat: signin & sign up page

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+.hero-banner-container .btn {
+  background-color: aqua;
+}
+
+.new-release {
+  /* padding: 2rem 6rem; */
+  /* border: 1px solid black; */
+}
+
```
Comment (AnkurChunekar) [inline style.css]: Remove unnecessary comments and classes whereever possible.
rest everything looks good.

**Review by AnkurChunekar — APPROVED** [review]

**Comment by AnkurChunekar** [comment]
Hey savita,
- I think you haven't included the confirm password and Name input field, please check LMS, it is mentioned there to have 2 fields for password and an input for user to enter name.
- Please write a small description for pr's.
- You can include Screenshots or CSB link for ref

File: style.css (hunk 1 above)
Comment (vermasavita) [reply style.css]: removed

**Comment by vermasavita** [comment]
@AnkurChunekar Hey Anuker, sure. I have added confirm password please check and approve. 

I'll keep in mind for other suggestions too.

**Review by AtulPant2704 — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+.new-release .new-release-head {
+  margin-top: 2rem;
+  padding: 0rem 2rem;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* margin: 1rem; */
```
Comment (Ayushi20-19) [inline style.css:77]: remove unwanted code

**Review by Ayushi20-19 — APPROVED** [review]

**Pull request merged**